users/common/strings.py

```diff
@@ -3,7 +3,7 @@
 
 from django.utils.translation import ugettext_lazy as _
 
-from users.common import constants
+from sheetstorm.settings import VALID_EMAIL_DOMAIN_LIST
```

Comment:
Please read https://github.com/Code-Poets/coding-style/wiki/python-style#django and fix the import.

Reply:
Thanks, @dybi has show me how to do this, but during implementation I forgot how to make this import. Done.